moped-etl/prefect/tasks/api.py

```diff
@@ -140,8 +142,8 @@ def create_moped_api_deploy_command(basename, config_secret_arn):
 
     # zappa deploy requires an active virtual environment
     # then use a subshell to zappa deploy in moped-api project folder
-    command = f"""python3 -m venv venv;
-    . venv/bin/activate;
+    command = f"""python3 -m virtualenv venv;
+    source venv/bin/activate;
```

Comment:
I don't see any harm in it, but I wonder why this needs to be executed in a subshell. This question should be taken in the context that I've never used subshell notation in bash scripts, but I don't see what environment changes or what-have-you need to be protected from leaking out of this part of the task. 

But anyway - all good! Just wanted to flag it for conversation or future learning on my part. 👍

Reply:
I'm glad that you checked in on this. This was born out of a Stack Overflow answer and looking back on it - there isn't a good reason for it. I'll update and test when I do the same for https://github.com/cityofaustin/atd-moped/pull/730#discussion_r929000845